### PR TITLE
add surplus to subst

### PR DESCRIPTION
**Pull request by peterstadler**

### CODE REVIEW

**Comment by peterstadler** [comment]
pinging @sydb and @ebeshero because I think this is good to go into the next release!

**Comment by ebeshero** [comment]
@peterstadler Looks good to me! Nice work with the Schematron.

**Comment by martinascholger** [comment]
@peterstadler - looks good to me. @sydb, do you approve this?

**Comment by peterstadler** [comment]
> In as far as it goes, this update seems absolutely fine. No reason not to merge. (And I am impressed that @peterstadler caught the needed change in expted_results/, too :-)

Thanks :)

> That said, wouldn’t it be nice to have at least one example that uses `<surplus>` in `<subst>`? 

I find it hard to invent a good example and none was provided at the ticket. We might ask for one but IMHO it's not a blocker.

> And maybe some prose describing what it’s for in PHSU (11.3.1.5, Substitutions)? 

Tried to do so in e83cc60, borrowing @cmm2209's words.

> And maybe change the `<desc>` of `<substJoin>` to mention superfluous text? (I also notice that description is missing its Oxford comma.) E.g.:
> > identifies a series of possibly fragmented additions, deletions, superfluous text, or other revisions on a manuscript that combine to make up a single intervention in the text

Thanks again. Duly added! And I also found it necessary to alter the  `<desc>` of `<subst>` in this direction.

@ebeshero and @sydb may I ask you once more to have a look at these changes?

**Comment by sydb** [comment]
> I find it hard to invent a good example and none was provided at the ticket. We might ask for one but IMHO it's not a blocker.

I agree it is not a blocker, but I don’t want us to forget, either. So my suggestion is to merge w/o example, but post a request for example(s) to the ticket (#1661) itself.

Will try to review prose soon. Thanks @peterstadler!

File: P5/Source/Guidelines/en/PH-PrimarySources.xml [hunk 1]
```diff
@@ -1326,6 +1326,13 @@ in the Eliot typescript mentioned above, there are --> several
                of the <soCalled>overlay</soCalled> levels implicit in the manuscripts of James
                Joyce's <title>Ulysses</title>. A fuller and more complex way of supporting such an
                approach is discussed in <ptr target="#PH-changes"/>. </p>
+            
+            <p>A special case of a substitution may consist of a superfluous word or phrase that is
```
Comment (sydb) [inline P5/Source/Guidelines/en/PH-PrimarySources.xml:1330]: Good. Suggested mildly modified version:
~~~xml
<p>A special case of a substitution may consist of a superfluous word
or phrase that is silently replaced by some addition. E.g. a scribe
abandons a word (without indicating it should be deleted), and then
writes a replacement word immediately after. Here the encoder may
interpret this as an <soCalled>unmarked</soCalled> deletion which can
then be combined with a corresponding addition to a substitution.</p>
~~~

**Comment by peterstadler** [comment]
revised the prose according to @sydb's proposal. @sydb what do you think of @ebeshero's [argument](#pullrequestreview-468017960) about the description of `<substJoin>`?

> However, I'm not sure I agree w/ @sydb that we needed to change anything in the desc for `substJoin.xml`. I think that one makes it unnecessarily difficult to puzzle through in the context of an element spec, _unless_ you're reading it in context with the Primary Sources chapter, as mostly I'm not sure people do. But doesn't the original desc carry the meaning we need for substJoin? I think the idea of superfluous text is implied sufficiently in "other revisions on a manuscript" besides additions / deletions.

**Comment by sydb** [comment]
Eh. I am not convinced either way (on whether `<desc>` of `<substJoin>` should mention superfluous text or not), so let’s leave it as is (as @ebeshero recommends) for now.

**Comment by peterstadler** [comment]
hopefully final ping for @ebeshero to have a look and merge if possible

**Pull request merged**